From: Kemeng Shi <shikemeng@huaweicloud.com>
Subject: mm/page_ext: remove unused return value of offline_page_ext
Date: Fri, 14 Jul 2023 19:47:47 +0800

Patch series "minor cleanups for page_ext".

This series contains some random minor cleanups for page_ext.  More
details can be found in respective patches.  


This patch (of 3):

offline_page_ext always returns 0 and no caller checks the return value. 
Just remove unused return value of offline_page_ext.

Link: https://lkml.kernel.org/r/20230714114749.1743032-1-shikemeng@huaweicloud.com
Link: https://lkml.kernel.org/r/20230714114749.1743032-2-shikemeng@huaweicloud.com
Signed-off-by: Kemeng Shi <shikemeng@huaweicloud.com>
Cc: Joonsoo Kim <iamjoonsoo.kim@lge.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/page_ext.c |    4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

--- a/mm/page_ext.c~mm-page_ext-remove-unused-return-value-of-offline_page_ext
+++ a/mm/page_ext.c
@@ -430,7 +430,7 @@ static int __meminit online_page_ext(uns
 	return -ENOMEM;
 }
 
-static int __meminit offline_page_ext(unsigned long start_pfn,
+static void __meminit offline_page_ext(unsigned long start_pfn,
 				unsigned long nr_pages)
 {
 	unsigned long start, end, pfn;
@@ -454,8 +454,6 @@ static int __meminit offline_page_ext(un
 
 	for (pfn = start; pfn < end; pfn += PAGES_PER_SECTION)
 		__free_page_ext(pfn);
-	return 0;
-
 }
 
 static int __meminit page_ext_callback(struct notifier_block *self,
_
